SevenSlots/SevenSlots/SevenSlots/Commands/RegisterCommand.cs

```diff
@@ -26,12 +26,31 @@ public bool CanExecute(object parameter)
             return true;
         }
 
-        public void Execute(object parameter)
+        public async void Execute(object parameter)
         {
-            userService.register(parameter as User);
-            string userString = JsonSerializer.Serialize(parameter as User);
-            Session.GeneralSettings = userString;
-            Application.Current.MainPage = new AppShell();
+            string message = await userService.register(parameter as User);
+
+            if (message == "Ok") {
+                string userString = JsonSerializer.Serialize(parameter as User);
+                Session.GeneralSettings = userString;
+                Application.Current.MainPage = new AppShell();
+            }
+            else if (message.Length > 30)
+            {
+                await App.Current.MainPage.DisplayAlert(
```

Comment:
Why write the DisplayAlerts so weirdly on many lines? Consider putting them on the same line.

Reply:
i think it's more readable like that!